tools/add_secrets.py

```diff
@@ -0,0 +1,56 @@
+#!/usr/bin/env python3
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+"""
+A script to help move a local copy of Fenix secrets into their usual destinations. This can
+be used to generate builds more closely mirroring release builds for performance testing.
+
+To use this, copy the YAML file defined in TaskCluster into a local file in the source root named
+.local_secrets.
+
+For example, the nightly secrets can be found here: https://firefox-ci-tc.services.mozilla.com/secrets/project%2Fmobile%2Ffenix%2Fnightly.local_secrets
+This may require requesting additional TaskCluster scopes.
+
+This script can then be invoked using:
+tools/add_secrets.py <release_type>
+"""
+
+import sys
+import yaml
+
+try:
+    build_type = sys.argv[1]
+except IndexError:
+    print(
+        """
+        Expected 1 argument defined build variant. Probably 'nightly', 'beta', or 'release'.
+        For example: python3 tools/add_secrets.py nightly
+        """
+    )
+    sys.exit(2)
+
+# These mappings are borrowed from taskcluster/transforms/build.py and should be kept in sync.
```

Comment:
I wonder if there is an easy way if we can share the mapping between the scripts so we don't have to manually keep them in sync 🤔

Reply:
I suppose a shared text file could be parsed between them. I'll consider if that's worth the effort when I come back to this PR